providers/sendgrid/src/lib/sendgrid.provider.ts

```diff
@@ -74,6 +77,26 @@ export class SendgridEmailProvider implements IEmailProvider {
     delete options.customData?.dynamicTemplateData;
     delete options.customData?.templateId;
 
+    const attachments = options.attachments?.map(
+      (attachment: IAttachmentOptions) => {
+        const attachmentJson: Partial<AttachmentJSON> = {
```

Comment:
here you have `Partial<AttachmentJSON>` but at the end you do `as AttachmentJSON`, so maybe define it from the beginning like that, or you have some issues?

Reply:
Good point i think I had some issues before we created the local `type AttachmentJSON`